src/content/experimental/ui-shell/usage.md

```diff
@@ -75,9 +104,17 @@ The L1 Menu is accessed by clicking on the header's left-side hamburger menu. It
 
 ![L1 Navigation menu](images/L1-navigation.png)
 
+-->
+
 ### L2 Side-nav
```

Comment:
I think we should actually remove the "L2" labels now that we're not doing the levels for the other components like "L0 Header". Keeping L2 in only for side-nav seems weird.